magento/graphql-ce#: Magento CLI. Support short commands name (#4908)
magento/devdocs: _includes/reference/cli-template.md

This reference contains {{ commands | size }} commands available through the `bin/magento` command-line tool.
The initial list is auto generated using `bin/magento list` command at the {{ site.data.var.ce }} edition.

{: .bs-callout-info }

> Magento CLI allows a short command name instead of a full one. For example, `bin/magento setup:upgrade` can be called using `bin/magento s:up`, `bin/magento s:upg`, etc. See [shortcut syntax](https://symfony.com/doc/current/components/console/usage.html#shortcut-syntax) to understand how to use shortcuts with any Magento CLI command.

Reviewer: ```suggestion
You can call Magento CLI commands using shortcuts instead of the full command name. For example, you can call `bin/magento setup:upgrade` using `bin/magento s:up`, `bin/magento s:upg`, etc. See [shortcut syntax](https://symfony.com/doc/current/components/console/usage.html#shortcut-syntax) to understand how to use shortcuts with any Magento CLI command.
```

Author: **done**

Thank you, @erikmarr
